addons/todo/views/project_task_views.xml

```diff
@@ -142,6 +144,40 @@
         </field>
     </record>
 
+    <!-- Todo conversion form (used by an action added in TodoFormView (contoller) -->
+    <record id="project_task_view_todo_conversion_form" model="ir.ui.view">
+        <field name="model">project.task</field>
+        <field name="name">project.task.view.todo.conversion.form</field>
+        <field name="priority">999</field>
+        <field name="arch" type="xml">
+            <form string="Convert to Task"
+                  js_class="todo_conversion_form">
+                <sheet>
+                    <group>
+                        <field name="project_id"
+                               domain="[('company_id', 'in', context.get('allowed_company_ids'))]"
+                               required="1"
+                               placeholder="Select an existing project"
+                               default_focus="1"/>
+                        <field name="user_ids"
+                               class="o_task_user_field"
+                               options="{'no_open': True, 'no_quick_create': True}"
+                               widget="many2many_avatar_user"
+                               domain="[('share', '=', False), ('active', '=', True)]"/>
+                        <field name="tag_ids" widget="many2many_tags"
+                               options="{'color_field': 'color', 'no_create_edit': True}"
+                               context="{'project_id': project_id}"
+                               placeholder="Choose tags from the selected project"/>
+                    </group>
+                </sheet>
+                <footer>
+                    <button name="action_convert_to_task" string="Convert to Task" type="object" class="btn-primary"/> 
+                    <button string="Cancel" class="btn-default" special="cancel"/>
```

Comment:
```suggestion
                    <button string="Discard" class="btn-default" special="cancel"/>
```
The class is really needed? 🧐

I see in the codebase, sometimes, we use `btn-secondary` and sometimes no class for this button. 🤔 

I change the label to `Discard` because now, the button is Discard in main form view and we had a task to rename `Cancel` into `Discard` in our apps. 🙂

Reply:
The class indeed does not seem needed. I've changed cancel to discard ;)